templates/shared/forms/_recaptcha.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script>
+  // move g-recaptcha-response to marketo field
+  var marketoForm = document.querySelector('form#mktoForm');
+  marketoForm.addEventListener(
+    'submit',
+    function() {
+      var input = document.createElement('input');
+      input.type = 'hidden';
+      input.name = "grecaptcharesponse";
+      input.value = grecaptcha.getResponse();
+      this.appendChild(input);
```

Comment:
In testing I realised that if you submit the form with errors, this gets run, `grecaptcharesponse` gets added, but then the form doesn't get submitted. If you then submit the form again, another `grecaptcharesponse` gets added, and an array of 2 values is sent to Marketo. I haven't checked, but I imagine Marketo won't handle that well.

So I suggest this:

``` javascript
  // move g-recaptcha-response to marketo field
  var marketoForm = document.querySelector('form#mktoForm');
  marketoForm.addEventListener(
    'submit',
    function() {
      var input = marketoForm.querySelector('[name=grecaptcharesponse]')
      if (! input) {
        input = document.createElement('input');
        input.type = 'hidden';
        input.name = "grecaptcharesponse";
        this.appendChild(input);
      }
      input.value = grecaptcha.getResponse();
    }
  )
```